lib/ai/client.go

```diff
@@ -77,3 +77,23 @@ func (client *Client) Summary(ctx context.Context, message string) (string, erro
 
 	return resp.Choices[0].Message.Content, nil
 }
+
+// ClassifyMessage takes a user message, a list of categories, and uses the AI mode as a zero shot classifier.
+func (client *Client) ClassifyMessage(ctx context.Context, message string, classes map[string]string) (string, error) {
```

Comment:
Can you add to the function description that the `classes` maps should have the category as the key and the description as the value?